cmake/DetectOptions.cmake

```diff
@@ -70,6 +70,14 @@ endfunction()
 # Multithreading
 find_package(Threads REQUIRED)
 
+# BigWhoop
+if(ADIOS2_USE_BigWhoop STREQUAL AUTO)
+ find_package(BWC CONFIG)
+elseif(ADIOS2_USE_BigWhoop)
+  find_package(BWC REQUIRED CONFIG)
+endif()
+set(ADIOS2_HAVE_BigWhoop ${BWC_found})
```

Comment:
This line does not work with my cmake. `BWC_found` is empty, however `BWC_FOUND` is set to 1 if BigWhoop is found.

Reply:
Thank you for your feedback. Indeed, there was a typo. I fixed it in a way that also conforms to the configuration of the corresponding CMake variable for other compression libraries. 

```
if(BWC_FOUND)
  set(ADIOS2_HAVE_BigWhoop TRUE)
endif()
```

I'd prefer the if statement so that the variable remains undeclared if BWC is not found. A query like `#ifdef ADIOS2_HAVE_BigWhoop` in a cpp source file would also fail without explicit value comparison with `TRUE`. And as said, it conforms with the solutions for similar use cases/compressors in `DetectOptions.cmake`.